src/scss/base/_reset.scss

```diff
@@ -3,9 +3,11 @@
   margin: 0;
   padding: 0;
 }
-
+html {
+  font-size: 62.5%;
```

Comment:
HTML and body should specific in pixel value instead

Reply:
Rem/em value will be calculated on upper element value in pixels

Reply:
By default, most browsers use a font size value of 16px, it mean 1rem=1em=16px.
html {
  font-size: 62.5%;
  /* changes a default 16px font size to 10px */
}
body{
  font-size: 1.6rem;
  /* font size = 16px */
}
Because rem reference the html element.

Reply:
`most browsers` means not all browser, the layout will be different between different browser, that's not the right way to implement UI, instead of using default value from browser, we need to set default value in our app instead.